AI-and-Analytics/Getting-Started-Samples/INC-Quantization-Sample-for-PyTorch/README.MD

```diff
@@ -64,6 +54,7 @@ The sample contains one Jupyter Notebook and one Python script.
 
 ## Run the `Getting Started with Intel® Neural Compressor for Quantization` Sample
```

Comment:
It is not clear what steps are for Offline Installer, what steps are for AI Tools Selector (not offline) and what steps are for both approaches. We need to revise document structure

Reply:
Is gitclone required for these samples?